### ISSUE TITLE
move the repository to GeoRust

### ISSUE THREAD

**Issue by antoine-de** [georust/rust-geos#3]
Hi,

we've done [some work](https://github.com/QwantResearch/rust-geos/) on your repository to add a [rust-geo](https://github.com/georust/rust-geo) conversion.

I think this project will have more visibility if it's moved to the [GeoRust](https://github.com/georust) organisation.

Would you be willing to move it ?

else I can either:
* move our fork in GeoRust, but then it would be marked as a fork thus It will be a bit more difficult for users to know it's the 'main' repository
* push our fork in GeoRust as a new project (and mentioning this repository in the readme), this way it won't be a fork, but I feel that it would be better for your original project to  become the official one

If you agree to move this repository, I'll ask the GeoRust guys to add you in the GeoRust organisation

Thanks for your great work,
Antoine

**Comment by mthh** [comment]
Hi,

I took a look and tried your fork: great job (which will also allow me to learn from the mistakes I made in my original code)!

Thank you for your proposal, I am totally inclined to move my directory to the GeoRust organization (by the way I am already a member of the organization but I don't know if I have the rights to add / move a repository).

I will try to move it and I'll let you know.

Thanks!
